src/help.sh

```diff
@@ -7,36 +7,38 @@ function kworkflow_help()
     '' \
     'The current supported targets are:' \
     '  Host - this machine' \
-    '  Qemu - qemu machine' \
     '  Remote - machine reachable via the network' \
     '' \
     'Commands:' \
-    '  backup - Save or restore kw data' \
-    '  bd - Build and install modules' \
+    '  init - Initialize kworkflow config file' \
     '  build,b - Build kernel' \
-    '  clear-cache - Clear files generated by kw' \
-    '  codestyle,c - Apply checkpatch on directory or file' \
-    '  config,g - Set kw config options' \
-    '  env - Handle kw envs' \
-    '  debug - Linux kernel debug utilities' \
     '  deploy,d - Deploy a new kernel image to a target machine' \
-    '  device - Show basic hardware information' \
+    '  bd - Build and install modules' \
     '  diff,df - Diff files' \
-    '  drm - Set of commands to work with DRM drivers ' \
-    '  explore,e - Explore string patterns' \
-    '  h,-h - Displays this help message' \
-    '  help,--help - Show kw man page' \
-    '  init - Initialize kworkflow config file' \
-    '  kernel-config-manager,k - Manage config files' \
-    '  mail - Send patches via email' \
+    '  ssh,s - SSH support' \
+    '  vars - Show variables' \
+    '  codestyle,c - Apply checkpatch on directory or file' \
+    '  self-update,u - kw self-update mechanism' \
     '  maintainers,m - Get maintainers and mailing list' \
-    '  man - Show manual pages' \
+    '  kernel-config-manager,k - Manage config files' \
```

Comment:
Don't know if this is redundant considering the feature name, but what do you say about `Manage kernel config files`?